modules/distribution-zones/src/test/resources/META-INF/services/org.apache.ignite.configuration.ConfigurationModule

```diff
@@ -1 +0,0 @@
-org.apache.ignite.internal.storage.impl.TestPersistStorageConfigurationModule
```

Comment:
What is this?Why did we remove this?

Reply:
Because of the test nature located in given module it's required to have both
```
testImplementation (project(':ignite-storage-page-memory'))
testImplementation(testFixtures(project(':ignite-storage-api')))
``` 
see
```
        clusterCfgMgr = new ConfigurationManager(
                List.of(DistributionZonesConfiguration.KEY),
                Set.of(),
                new TestConfigurationStorage(DISTRIBUTED),
                List.of(),
                List.of(PersistentPageMemoryDataStorageConfigurationSchema.class)
        );
```

With the default provided in META-INF it's not possible to include `ignite-storage-...` hierarchy, following exception will be thrown:
```
case already exists for value [aipersist]
java.lang.IllegalStateException: case already exists for value [aipersist]
	at com.facebook.presto.bytecode.BytecodeUtils.checkState(BytecodeUtils.java:89)
	at org.apache.ignite.internal.configuration.asm.StringSwitchBuilder.addCase(StringSwitchBuilder.java:104)
```
That's why given resource file was removed.